cert/cert.go

```diff
@@ -120,25 +130,51 @@ func (ca *CA) Refresh() (bool, error) {
 		return false, err
 	}
 
-	if bytes.Equal(cert, ca.pem) {
+	isCACertSame, err := CompareCertificates(cert, ca.pem)
+	if err != nil {
+		log.Infof("cert: error comparing CA certificates")
```

Comment:
This is probably a warning or error at minimum

Reply:
you're right, i'll make it a warning